src/pool/srv_cli.c

```diff
@@ -29,6 +29,9 @@ dsc_pool_close(daos_handle_t ph)
 		return 0;
 
 	pl_map_disconnect(pool->dp_pool);
+	dc_pool_hdl_unlink(pool); /* -1 ref from dc_pool_hdl_link(pool); */
+	dc_pool_put(pool);	  /* -1 ref from dc_pool2hdl(pool, ph); */
```

Comment:
not very sure for above two, but  should correspond to ref taken in dsc_pool_open(), in that func it only take 1 extra ref (because there is one dc_pool_put at the end of dsc_pool_open). and seems different in "if (daos_handle_is_valid(*ph))" case.
maybe better to confirm with @liw

Reply:
@wangdi1 wrote this code. The "valid(*ph)" case in dsc_pool_open is indeed confusing. AFAICS, Dmitry's change here is correct.